pkg/importer/BUILD.bazel

```diff
@@ -93,7 +97,6 @@ go_test(
         "//tests/utils:go_default_library",
         "//vendor/cloud.google.com/go/storage:go_default_library",
         "//vendor/github.com/aws/aws-sdk-go/service/s3:go_default_library",
-        "//vendor/github.com/containers/image/v5/types:go_default_library",
```

Comment:
This is the line the verify is complaining about. It wants to add it back again. I take it this is not available in s390x?

Reply:
When I ran `git difftool -y --extcmd=hack/diff-csv.sh` manually it found an issue with the indentation in 2 lines in `tools/cdi-func-test-registry-init/BUILD.bazel` as well.